README.md

```diff
@@ -14,6 +14,18 @@ SPDX-License-Identifier: Apache-2.0
 
 Service to validate SCL Files.
 
+## Rest and Websockets versions
+
+The validation can be done using both Rest and Websockets as transportation technic.
```

Comment:
Never used technic, I always use technique.
According to Google Translate, it's correct! I learned something today!

Reply:
So, do we need to change it :-)